CONTRIBUTING.md

```diff
@@ -231,17 +231,30 @@ From the RMarkdown Cookbook [bibliographies chapter](https://bookdown.org/yihui/
 You can obtain formatted `LaTeX` reference entry following the instructions below and copying and pasting the whole thing in `reference.bib`.
 The references in `reference.bib` should be kept in alphabetical order (this will reduce the chances of adding duplicates).
 
+Automatic reference renders (like those recommended below) don't maintain consistent syntax (though they usually will still work).
+You may/will still need to do some manual edits to maintain consistency with the current `references.bib`-listed sources.
+
+**Adding sources rules:**  
+
+- All sources need to have URLs.
+- Where applicable, sources should have doi's.
+- `{}` are preferred to `" "`.
+- To preserve capitalization, use an extra `{}` around what should be capitalized.
+
+**How to add sources by type**  
+
 - _R packages_: Use this kind of command in the console: `toBibtex(citation("tidyverse"))`.
 Copy and paste the output to the `references.bib` file.
 
-- _Articles_: For PubMed articles, you can use [this website to find the article](https://www.bioinformatics.org/texmed/). Select the correct article that pops up and click `Export`. Then copy and paste the reference to the `references.bib` file.
+- _Articles_: For articles, you can use [this website to get a BibTex](https://www.doi2bib.org/ ).
+Then copy and paste the reference to the `references.bib` file.
 
 - _Websites_: Websites can be added using this template:
 ```
 @online{website,
```

Comment:
I think this should be: 

```suggestion
@website{citekey,
```

Also, in sections below, we should use `@article` lower case and `@website` for consistency.

Reply:
Ah. I didn't know it could be lower case but that makes sense.